ui/raidboss/data/06-ew/raid/p10n.txt

```diff
@@ -1,6 +1,81 @@
 ### P10N: Anabaseios: The Tenth Circle
+#
+# -ii 825E 825F 8261 8263 8277 8279 825B 825C 8266 8269 8271 8273 8274 826B 826C 826D 826E 86B5 86B6 86B7 86B8
+#
+# Ignored Ids:
+# 825[EF] - 'Bury' action/damage from Pandaemoniac Pillars
+# 8261      'Imprisonment' extra
+# 8263      'Cannonspawn' extra
+# 8277      'Pandaemonic Meldown' extra
+# 8279      'Soul Grasp' extra
+# 825[BC]   'Entangling Web' extra
+# 8266      'Pandaemonic Ray' extra
+# 8269      'Touchdown' extra
+# 8271      'Parted Plumes' blades
+# 827[34]   'Wicked Step' individual towers
+# 826[B-E]  Supplemental AoE damage from 'Harrowing Hell'
+# 86B[5-8]  Supplemental AoE damage from 'Harrowing Hell'
+
+# Note: The boss's name has a Unicode character (�)  which is causing syncs to fail.
+# The character is manually escaped in the syncs with the correct codepoint (U+00E6).
+# Entry names use "ae" as separate characters to ensure proper UI display.
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+2.5 "--sync--" sync / 1[56]:[^:]*:Pand\u00e6monium:827A:/
+10.9 "Silkspit (marked)" sync / 1[56]:[^:]*:Pand\u00e6monium:8259:/
+23.0 "Silkspit (resolves)" sync / 1[56]:[^:]*:Pand\u00e6monium:825A:/
+35.4 "Pandaemoniac Pillars" sync / 1[56]:[^:]*:Pand\u00e6monium:825D:/
+43.4 "Imprisonment" sync / 1[56]:[^:]*:Pand\u00e6monium:8262:/
+50.1 "Ultima" sync / 1[56]:[^:]*:Pand\u00e6monium:827B:/
+59.9 "Pandaemoniac Pillars" sync / 1[56]:[^:]*:Pand\u00e6monium:825D:/
+67.8 "Cannonspawn" sync / 1[56]:[^:]*:Pand\u00e6monium:8264:/
+74.6 "--sync--" sync / 1[56]:[^:]*:Pand\u00e6monium:87A2:/
+80.0 "Pandaemoniac Meltdown" sync / 1[56]:[^:]*:Pand\u00e6monium:8276:/
+90.1 "Soul Grasp" sync / 1[56]:[^:]*:Pand\u00e6monium:8278:/
+101.3 "Entangling Web" sync / 1[56]:[^:]*:Pand\u00e6monium:8721:/
+114.7 "Silkspit (marked)" sync / 1[56]:[^:]*:Pand\u00e6monium:8259:/
+126.7 "Silkspit (resolves)" sync / 1[56]:[^:]*:Pand\u00e6monium:825A:/
+140.4 "Pandaemoniac Ray East/West" sync / 1[56]:[^:]*:Pand\u00e6monium:826[57]:/
```

Comment:
I think the "East/West" isn't necessary here if it's just called "Pandaemoniac Ray" in game (I think?).  If there were North/South or In/Out or other variations that would make sense to differentiate in the timeline then extra text is good, but otherwise just adds more non-automatic translation work.